src/components/common/WalletInfo/index.tsx

```diff
@@ -19,17 +23,18 @@ const WalletInfo = ({ wallet, chain }: { wallet: ConnectedWallet; chain: ChainIn
       </Box>
       <Box>
         <Typography variant="caption" component="div" className={css.walletDetails}>
-          {wallet.label} @ {chain.chainName}
+          {wallet.label} @ {walletChain?.chainName || UNKNOWN_CHAIN_NAME}
         </Typography>
         <Typography variant="caption" fontWeight="bold" component="div">
           {wallet.ens ? (
             <div>{wallet.ens}</div>
           ) : (
             <EthHashInfo
-              prefix={chain.shortName}
+              prefix={walletChain?.shortName || ''}
               address={wallet.address}
               showName={false}
               showAvatar
+              showPrefix={!walletChain}
```

Comment:
I think that this `!walletChain` flag is removing always the prefix in the address

![Captura de pantalla 2023-01-27 a las 16 13 34](https://user-images.githubusercontent.com/26763673/215120549-2f398776-7146-40f2-a9f9-094b2e2ed289.png)
![Captura de pantalla 2023-01-27 a las 16 13 42](https://user-images.githubusercontent.com/26763673/215120555-12d3c54e-acb9-49dd-8791-6553d610521f.png)

Reply:
Nice catch. Remnants from before the refactor 🙈. I've fixed it.